test/unittests/utils/xml_parsing/test_get_parsed_resources.py

```diff
@@ -34,6 +35,17 @@
 }
 
 
+@pytest.mark.parametrize(
+    ("input_str", "expected"),
+    [
+        ("https://app.dasch.swiss", "http://app.dasch.swiss"),
```

Comment:
I know it doesn't matter, but to be more realistic:

```suggestion
        ("https://api.stage.dasch.swiss/ontology/0854/metadata/v2#Archive", "http://api.stage.dasch.swiss/ontology/0854/metadata/v2#Archive"),
```

Reply:
Thanks for spotting that.